test/integration/dev_tokens_ft_tests.js

```diff
@@ -22,6 +22,7 @@ var aCorrelationHeader = {
 }
 
 var connectorMock = nock(process.env.CONNECTOR_URL, aCorrelationHeader)
+var directDebitConnectorMock = nock(process.env.DIRECT_DEBIT_CONNECTOR_URL, aCorrelationHeader)
```

Comment:
If you are using nock, you need to make sure you call `'nock.cleanAll'` after

Reply:
Isn't it enough to do that in: ```  afterEach(function () {
      nock.cleanAll()
      app = null
    })
```?

Reply:
sorry didnae see it